plexupdate.sh

```diff
@@ -275,7 +275,7 @@ fi
 
 # Remove any ~ or other oddness in the path we're given
 DOWNLOADDIR="$(cd ${DOWNLOADDIR// /\\ } && pwd)"
-if [ -z "${DOWNLOADDIR}" ]; then
+if [ ! -d "${DOWNLOADDIR}" ]; then
```

Comment:
Fair enough, tested it too after I wrote the comment and ! -d "" will fail, so all good.